File: app/src/main/java/com/mobillium/paparasdk/sdk/sampleapp/webservice/models/AllowedPaymentType.java
package com.mobillium.paparasdk.sdk.sampleapp.webservice.models;

/**
 * Created by oguzhandongul on 15/09/2017.
 */

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class AllowedPaymentType {

    @SerializedName("paymentMethod")
    @Expose
    private String paymentMethod;
    @SerializedName("id")
    @Expose
    private String id;
    @SerializedName("createdAt")
    @Expose
    private String createdAt;

    public String getPaymentMethod() {
        return paymentMethod;
    }

    public void setPaymentMethod(String paymentMethod) {
        this.paymentMethod = paymentMethod;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getCreatedAt() {
        return createdAt;
    }

    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

}

File: app/src/main/java/com/papara/sdk/sampleapp/data/models/AllowedPaymentType.kt
package com.papara.sdk.sampleapp.data.models

import com.google.gson.annotations.SerializedName

data class AllowedPaymentType(
    @SerializedName("paymentMethod")
    val paymentMethod: String? = null,

    @SerializedName("id")
    val id: String? = null,

    @SerializedName("createdAt")
    val createdAt: String? = null
)
